README.md

```diff
@@ -1,15 +1,16 @@
 # redyt
-Search reddit with CLI
+Scrape Reddit images from the Terminal
 
-This script will automatically create the two following folders:
+This shell script will automatically create the two following folders:
   - `~/.cache/redyt`
```

Comment:
Please consider replacing this location with /tmp/redyt. Benefit is that many users have /tmp stored on tmpfs and that will reduce use of ssd storage and maybe speed write up

Reply:
Seems reasonable to me. Has been added in https://github.com/Bugswriter/redyt/pull/4/commits/4c55319fe5e544e4371f6c95cd3345b3c96f14d8.